spec/requests/content_blocks_spec.rb

```diff
@@ -15,6 +15,8 @@ def compare_content_blocks(object, content_block)
   end
 
   context '#index' do
+    before(:each) { Rails.cache.clear }
```

Comment:
then you are not doing it, because all the tests were green when we disabled the cache :)